kong/plugins/aws-lambda/aws-serializer.lua

```diff
@@ -81,6 +86,39 @@ return function(ctx, config)
   local uri = var.upstream_uri or var.request_uri
   local path = uri:match("^([^%?]+)")  -- strip any query args
 
+  local requestContext
+  do
+    local http_version = ngx_get_http_version()
+    local protocol = http_version and 'HTTP/'..http_version or nil
+    local httpMethod = var.request_method
+    local domainName = var.host
+    local domainPrefix = split(domainName, ".")[1]
+    local identity = {
+      sourceIp = var.remote_addr,
+      userAgent = headers["user-agent"],
+    }
+    local requestId = var.request_id
+    local start_time = ngx_req_start_time()
+    local requestTime = date(start_time):fmt("${iso}")
+    local requestTimeEpoch = start_time * 1000
+
+    -- Kong does not have the concept of stage, so we just let resource path be the same as path
+    local resourcePath = path
+
+    requestContext = {
```

Comment:
why the locals, most or not used, we can add the values in-line to the table below?

(nitpick: feel free to ignore)